handsontable/types/helpers/mixed.d.ts

```diff
@@ -1,4 +1,5 @@
 export function stringify(value: any): string;
+export function isBoolean(value: string | boolean): boolean;
```

Comment:
I'd change the type of input argument to `any`
```suggestion
export function isBoolean(value: any): boolean;
```